packages/api-server/api_server/models/tortoise_models/tasks.py

```diff
@@ -82,4 +82,7 @@ class TaskFavorite(Model):
     user = CharField(255, null=False, index=True)
 
 
+TaskQueueEntryPydantic = pydantic_model_creator(
+    TaskState, exclude=("data", "category", "unix_millis_warn_time")
+)
```

Comment:
I think we should start avoiding use of `pydantic_model_creator` as it is not type safe. It's a bit troublesome to write it by hand but I think it is worth it. In the future we can consider a new generator that keep typings.

Reply:
Good call, I've created a separate model for returning